llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -244,6 +244,10 @@ def FeatureExperimentalZeroingPseudos
 def FeatureUseScalarIncVL : SubtargetFeature<"use-scalar-inc-vl",
   "UseScalarIncVL", "true", "Prefer inc/dec over add+cnt">;
 
+def FeatureUseFixedOverScalableIfEqualCost: SubtargetFeature<"use-fixed-over-scalable-equal-cost",
```

Comment:
Perhaps I'm being a bit pedantic here (tell me if I am!), but I think it makes sense to have the flag match feature name, so that the flag should be "use-fixed-over-scalable-if-equal-cost". My thinking is that it's only 3 extra characters in an already long name, and is a bit clearer/consistent. :)